lib/src/fairing/fairings.rs

```diff
@@ -110,7 +115,11 @@ impl Fairings {
         }
 
         if !self.all_fairings.is_empty() {
-            info!("{}{}:", Paint::masked("📡  "), Paint::purple("Fairings"));
+            if self.use_emoji {
```

Comment:
These `self.use_emoji` checks repeated everywhere add a lot of redundancy.

Can Paint be extended to have this "use-emoji-or-not-aware" capability, *in addition to* Paint::masked? (@SergioBenitez) I see the value in treating emoji use and color orthogonally, but I don't like it duplicated at each log invocation.